src/cdk/overlay/overlay-directives.ts

```diff
@@ -181,6 +181,9 @@ export class CdkConnectedOverlay implements OnDestroy, OnChanges {
   /** Whether the overlay is open. */
   @Input('cdkConnectedOverlayOpen') open: boolean = false;
 
+  /** Whether the overlay is open. */
+  @Input('cdkConnectedOverlayCloseOnEscape') closeOnEscape: boolean = true;
```

Comment:
We have something similar in the dialog, bottom sheet and sidenav called `disableClose` which we should probably align with. Note that in the other components it means disabling close when pressing escape and clicking on the backdrop.

Reply:
+1, we should be consistent with the name and the behavior

Reply:
Didn't know about `disableClose`, totally agree with it, I will modify PR soon

Reply:
@crisbeto I refactored code, also align behavior as it is in Dialog.
But `overlay-directive` doesn't have close action on backdrop [click](https://github.com/angular/components/blob/f21acde8341ecff7731aeac044f8ced3b1e08459/src/cdk/overlay/overlay-directives.ts#L394), I verified it manually also created [test](https://github.com/angular/components/pull/20585/commits/ab9d94fd4e82e060d9e180123e56c3e89dffb298#diff-2988ed2cf46548b0ded8e4be73b86babR178), that shows current behavior